QuasarHelios2015/src/org/team1540/quasarhelios/Elevator.java

```diff
@@ -42,7 +42,8 @@
     public static final BooleanInput atTop = atTopStatus;
     public static final BooleanInput atBottom = atBottomStatus;
 
-    private static FloatInput winchSpeed = ControlInterface.mainTuning.getFloat("Elevator Winch Speed +M", 1.0f);
+    private static FloatInput winchRaisingSpeed = ControlInterface.mainTuning.getFloat("Elevator Winch Raising Speed  +M", 1.0f);
+    private static FloatInput winchLoweringSpeed = ControlInterface.mainTuning.getFloat("Elevator Winch Lowering Speed  +M", 1.0f);
```

Comment:
Why two spaces? Also, it might be more consistent to have "Speed Raising" instead (for hierarchical sorting)